stacks/cli/src/commands/function/dependency/install.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  Command,
+  CommandMetadata,
+  CommandMetadataInput,
+  CommandMetadataOption,
+  CommandLineInputs,
+  CommandLineOptions,
+  validators
+} from "@ionic/cli-framework";
+
+import * as authentication from "../../../authentication";
+import * as request from "../../../request";
+import {Function} from "../../../interface";
+import {Logger} from "../../../logger";
+
+export class InstallCommand extends Command {
+  logger?: Logger = new Logger(process.stdout, process.stderr);
+  authentication = authentication;
+  request = request;
+
+  token;
+  server;
+
+  async getMetadata(): Promise<CommandMetadata<CommandMetadataInput, CommandMetadataOption>> {
+    return {
+      name: "install",
+      summary: "Install npm-package to function.",
+      inputs: [
+        {
+          name: "package-name",
+          summary: "The name of npm-package that will be installed.",
+          validators: [validators.required]
+        }
+      ],
+      options: [
+        {
+          name: "all",
+          summary: "Install specified npm-package to each function.",
+          type: Boolean
+        }
+      ]
+    };
+  }
+
+  showLoginError() {
+    this.logger.error("You need to login before start this action. To login: ");
+    this.logger.info("spica login <username> <password>");
```

Comment:
When you extend this namespace from `SpicaNamespace` you will be able to change these lines to 
```suggestion
    this.namespace.logger.error("You need to login before start this action. To login: ");
    this.namespace.logger.info("spica login <username> <password>");
```